format-avro/src/main/java/io/cdap/plugin/format/avro/AvroToStructuredTransformer.java

```diff
@@ -35,6 +38,7 @@
 public class AvroToStructuredTransformer extends RecordConverter<GenericRecord, StructuredRecord> {
 
   private final Map<Integer, Schema> schemaCache = Maps.newHashMap();
+  private final Map<String, JsonObject> customTypes = Maps.newHashMap();
```

Comment:
As soon as you do the resolution, please implement it with namespace support, see https://avro.apache.org/docs/current/spec.html#names